rhino/src/main/java/org/mozilla/javascript/NativeSymbol.java

```diff
@@ -23,13 +23,21 @@ public class NativeSymbol extends IdScriptableObject implements Symbol {
     private static final Object GLOBAL_TABLE_KEY = new Object();
     private static final Object CONSTRUCTOR_SLOT = new Object();
 
+    static final SymbolKey GETDESCRIPTION = new SymbolKey("[Symbol.getDescription]");
+
     private final SymbolKey key;
     private final NativeSymbol symbolData;
 
     public static void init(Context cx, Scriptable scope, boolean sealed) {
         NativeSymbol obj = new NativeSymbol("");
         ScriptableObject ctor = obj.exportAsJSClass(MAX_PROTOTYPE_ID, scope, false);
 
+        ScriptableObject desc = (ScriptableObject) cx.newObject(scope);
+        desc.put("enumerable", desc, Boolean.FALSE);
+        desc.put("configurable", desc, Boolean.TRUE);
+        desc.put("get", desc, obj.get(GETDESCRIPTION, obj));
```

Comment:
```suggestion
        desc.put("get", desc, new LambdaFunction(scope, 0, NativeSymbol::js_getDescription));
```
I was able to change the code to look like this to not have to deal with the new SymbolKey and without having to deal with converting everything else to using lambda. Is this good?

Reply:
I think that'll do, but would like @gbrail to have a look as well